### PR TITLE
[docs] Docs on DeckhouseReleases manifests generated by `dhctl mirror`

### PR DESCRIPTION
**Pull request by mvasl**
## Description

Documented usage of the DeckhouseReleases manifests that `dhctl mirror` command generates during pulls and fixed some small inconsistencies around environment variables and CLI flags usage

## Why do we need it, and what problem does it solve?

Our doc lacks any mentions of these generated manifests and they are somewhat important.
Closes https://github.com/deckhouse/deckhouse/issues/3313

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Documented usage of the DeckhouseReleases manifests that dhctl mirror generates during pulls.
impact_level: low
```

### CODE REVIEW

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 1]
```diff
@@ -483,14 +483,20 @@ This feature is available in Enterprise Edition only.
    > Please note that the images will be uploaded to the registry along the path specified in the `--registry` parameter (in the example above - /deckhouse/ee).
    > Before running the command, make sure this path exists in your registry, and the account you are using has write permissions.
 
-   If your registry does not require authentication, omit both `--registry-login` and `--registry-password` flags.
+   If your registry does not require authentication, omit both `--registry-login` and `--registry-password` flags or `DHCTL_CLI_MIRROR_USER`/`DHCTL_CLI_MIRROR_PASS` variables.
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:486]:
```suggestion
   If your registry does not require authentication, you may omit both `--registry-login` and `--registry-password` flags as well as `DHCTL_CLI_MIRROR_USER`/`DHCTL_CLI_MIRROR_PASS` variables.
```

Comment (kirkonru) [reply docs/documentation/pages/DECKHOUSE-FAQ.md:486]: "Or" in this case may be interpreted as "omit the variables but specify the flags" or vice versa. What we mean to say is that specifying them is not necessary, right?

**Review by kirkonru — APPROVED** [review]

File: docs/documentation/pages/DECKHOUSE-FAQ.md (hunk 1 above)
Comment (mvasl) [reply docs/documentation/pages/DECKHOUSE-FAQ.md]: Exactly

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 2]
```diff
@@ -417,13 +417,13 @@ Deckhouse поддерживает работу только с Bearer token-с
...
-1. Запустите установщик Deckhouse версии 1.56.3 или выше.
+1. Запустите установщик Deckhouse версии 1.58.6 или выше.
 
    ```shell
-   docker run -ti --pull=always -v $(pwd)/d8-images:/tmp/d8-images registry.deckhouse.ru/deckhouse/ee/install:v1.58.4 bash
+   docker run -ti --pull=always -v $(pwd)/d8-images:/tmp/d8-images registry.deckhouse.ru/deckhouse/ee/install:v1.58.6 bash
    ```
 
-   Обратите внимание, что в контейнер установщика монтируется директория с файловой системы хоста, в которую будут загружены образы Deckhouse.
+   Обратите внимание, что в контейнер установщика монтируется директория с файловой системы хоста, в которую будут загружены образы Deckhouse и сгенерированы манифесты [DeckhouseReleases](https://deckhouse.ru/documentation/v1/modules/002-deckhouse/cr.html#deckhouserelease).
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:426]:
```suggestion
   Обратите внимание, что в контейнер установщика монтируется директория с файловой системы хоста, в которую будут загружены образы Deckhouse и сгенерированы манифесты [DeckhouseReleases](modules/002-deckhouse/cr.html#deckhouserelease).
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 3]
```diff
@@ -416,13 +416,13 @@ This feature is available in Enterprise Edition only.
...
-1. Run the Deckhouse installer version 1.56.3 or higher.
+1. Run the Deckhouse installer version 1.58.6 or higher.
 
    ```shell
-   docker run -ti --pull=always -v $(pwd)/d8-images:/tmp/d8-images registry.deckhouse.io/deckhouse/ee/install:v1.58.4 bash
+   docker run -ti --pull=always -v $(pwd)/d8-images:/tmp/d8-images registry.deckhouse.io/deckhouse/ee/install:v1.58.6 bash
    ```
 
-   Note that the directory on the host will be mounted in the installer container. This directory will contain the pulled Deckhouse tarball.
+   Note that the directory on the host will be mounted in the installer container. This directory will contain the pulled Deckhouse tarball and generated [DeckhouseReleases](https://deckhouse.io/documentation/v1/modules/002-deckhouse/cr.html#deckhouserelease) manifests.
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:425]:
```suggestion
   Note that the directory on the host will be mounted in the installer container. This directory will contain the pulled Deckhouse tarball and generated [DeckhouseReleases](modules/002-deckhouse/cr.html#deckhouserelease) manifests.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [7953](https://github.com/deckhouse/deckhouse/pull/7953) to the branch [release-1.58](https://github.com/deckhouse/deckhouse/tree/release-1.58) successful!

**Pull request merged**